<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>urunGirisi</class>
 <widget class="QWidget" name="urunGirisi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tbwUrun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>271</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEkle">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>83</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>EKLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ürün</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ürünler</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtUrun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSil">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>83</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>SİL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
